audiorecorder/src/main/AndroidManifest.xml

```diff
@@ -2,12 +2,17 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.odk.collect.audiorecorder">
 
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
```

Comment:
This is only required on Android 28+ as far as I know.

Reply:
Looks like it's based on the targetSdk version.

Reply:
Ah right I always get confused about minimum API levels and when target changes the behaviour.